static/js/rtl.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns true if the given language is written from right to left (requires rtl css)
+ * 
+ * @param {string} locale 
+ * @returns {boolean}
+ */
+module.exports = function isRTL (locale) {
```

Comment:
maybe this can go into static/webpack since it's only used as part of the build? right now static/js is for stuff that goes on the browser